components/Pill/Pill.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Label } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import styles from './Pill.less';
+
+export const Pill = props => {
+  const { circular, color, customClass, size, text } = props;
+
+  return (
+    <div className="label-container">
+      <Label
+        className={`${styles.pill} ${styles[`${customClass}`]}`}
```

Comment:
Can use `import { classNames } from '../../../utilities/css';` to join classnames easily, other than that LGTM!